src/style.css

```
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  display: block;
+  margin: 0;
+  height: 40px;
+}
+
+.nav {
+  height: inherit;
+  margin: 0;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  height: inherit;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin: 0 20px 0 0;
+  height: 100%;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

1) use margin-right instead of margin: 0 20px 0 0;
2) you can simplify a little 
```suggestion
.nav__item {
  display: flex;
  align-items: center;
  height: 100%;
}

.nav__item:not(last-child) {
  margin-right: 20px;
}
```